### ISSUE TITLE
vimtex seems to conflict with vim-polyglot

### ISSUE THREAD

**Issue by aouelete** [lervag/vimtex#450]
I am using vimtex with neovim. Recently I noticed that running compiling LaTeX documents while running the [vim-polyglot](https://github.com/sheerun/vim-polyglot) plugin causes the following errors:
```
Error detected while processing function <SNR>45_fix_quickfix_paths:
line   13:
E121: Undefined variable: s:qf_root
E116: Invalid arguments for function simplify(s:qf_root . '/' . bufname(l:qf.bufnr)), ':.')
E116: Invalid arguments for function fnamemodify(simplify(s:qf_root . '/' . bufname(l:qf.bufnr)), ':.')
E15: Invalid expression: fnamemodify(simplify(s:qf_root . '/' . bufname(l:qf.bufnr)), ':.')
Error detected while processing function LatexBox_Latexmk:
line  210:
E171: Missing :endif
```

### Minimal document
For some reason the issue was not reproducible when `\maketitle` was not present.
```tex
\documentclass{minimal}
\begin{document}

\author{Me}
\maketitle

Hello World!

\end{document}
```

### Minimal vimrc file

```vim
set nocompatible

" Load Vimtex
let &rtp  = '~/.config/nvim/plugged/vimtex,' . &rtp
let &rtp .= ',~/.config/nvim/plugged/vimtex/after'

" Load polyglot
let &rtp = '~/.config/nvim/plugged/vim-polyglot,' . &rtp

filetype plugin indent on
syntax enable
```

**Comment by lervag** [comment]
You should _not_ use vimtex with vim-polyglot! The latter bundles LaTeX-Box, which is not compatible. Thus, if you really want to use vim-polyglot, I suggest that you add the following to your vimrc file:

```vim
let g:polyglot_disabled = ['latex']
```

**Comment by aouelete** [comment]
Ok, thanks for clarifying.
On May 2, 2016 1:53 AM, "Karl Yngve Lervåg" <notifications@github.com>
wrote:

Closed #450 <https://github.com/lervag/vimtex/issues/450>.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/lervag/vimtex/issues/450#event-647140573>
